nav2_behavior_tree/plugins/action/controller_selector_node.cpp

```diff
@@ -36,16 +36,25 @@ ControllerSelector::ControllerSelector(
 
   getInput("topic_name", topic_name_);
 
+  rclcpp::QoS qos(rclcpp::KeepLast(1));
+  qos.transient_local().reliable();
```

Comment:
When you have the PR for the goal checker docs, make sure to see that there expecting the QoS to be reliable and transient local. Add that to the other selector configuration pages as well so its not a surprise to folks when things don't connect as expected without changing QoS